scss/components/_activity-filter.scss

```diff
@@ -158,3 +158,29 @@
     margin-top: 5px;
   }
 }
+
+@media (max-width: 1420px) {
```

Comment:
Why are we having this media query here? Can we add something about this in the PR description.

Also, let's document this breakpoint in the _variable.scss files.